src-tauri/tauri.conf.json

```diff
@@ -51,7 +51,7 @@
       "csp": null
     },
     "updater": {
-      "active": false,
+      "active": true,
```

Comment:
I have a PR up that disables the updater in CI for builds by pushes. You can merge it early if you wanna give it a try to confirm it works ^^

Reply:
thanks done, not very experienced with gh CI but ill figure it out.